queue_services/entity-pay/src/entity_pay/worker.py

```diff
@@ -137,11 +137,10 @@ def get_filing_by_payment_id(payment_id: int) -> Filing:
     except OperationalError as err:
         logger.error('Queue Blocked - Database Issue: %s', json.dumps(payment_token), exc_info=True)
         raise err  # We don't want to handle the error, as a DB down would drain the queue
-    except FilingException as err:
-        logger.error('Queue Error - cannot find filing: %s'
-                     '\n\nThis message has been put back on the queue for reprocessing.',
+    except FilingException:
+        # log to sentry and absorb the error, ie: do NOT raise it, otherwise the message would be put back on the queue
+        logger.error('Queue Error - cannot find filing: %s',
```

Comment:
Was this tested to ensure it makes it into Sentry?
I know capture_message posts to Sentry, and unhandled errors roll up to Sentry, but I haven't checked that the logger sends any/all logger.error messages to Sentry and not just to stderr.

Reply:
Ready for review again, thanks for catching that.